packages/odyssey/src/scss/components/_status.scss

```diff
@@ -0,0 +1,63 @@
+.ods-status {
+  margin-bottom: $base-spacing;
+
+  &:last-child {
+    margin-bottom: 0;
+  }
```

Comment:
This is interesting, why do we need it?

Reply:
We do something similar for a few components. If our component is inside of a container with padding, we want it to sit flush with the bottom rather than have additional spacing beneath it.

Reply:
Just realized I need to move this to a layout file as well for WC friendliness.